src/CalcViewModel/CalcViewModel.vcxproj

```diff
@@ -40,6 +40,7 @@
     <RootNamespace>CalcViewModel</RootNamespace>
     <DefaultLanguage>en-US</DefaultLanguage>
     <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+    <PreferredToolArchitecture>x64</PreferredToolArchitecture>
```

Comment:
According to the [docs](https://docs.microsoft.com/en-us/cpp/build/walkthrough-using-msbuild-to-create-a-visual-cpp-project), this is supposed to appear _after_ the import of `Microsoft.Cpp.default.props` and here you have it before.  You can take a look at how CalcManager sets this up:
https://github.com/Microsoft/calculator/blob/ad25feda6bfac4b4779b92500b01d341931dcbb0/src/CalcManager/CalcManager.vcxproj#L51-L55